src/infobar.js

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react';
+import {
+  AppRegistry,
+  StyleSheet,
+  Text,
+  Image,
+  View
+} from 'react-native';
+
+export default class InfoBar extends Component {
+  constructor(props) {
+    super(props);
+  }
+  renderPicture(){
+    return(
+      <View style={styles.picture}>
+        <Image source={require('./img/bogdan.jpg')} style={{height: 30, width: 30, borderRadius: 30/2,}}/>
+      </View>
+    );
+  }
+  renderName(){
+    return(
+        <Text style={styles.nume}>Olaru Bogdan</Text>
+    );
+  }
+  renderUserInfo(){
+    return(
+      <View style={styles.userInfo}>
+        {this.renderPicture()}
+        {this.renderName()}
+      </View>
+    );
+  }
+  renderTime(){
+    return(
+      <Text style={styles.time}>
+        2:30
```

Comment:
We need to make this time tick, change every second. We can use the JS setInterval & clearInterval function to do this, you can find an example here: https://stackoverflow.com/questions/41294576/react-native-show-current-time-and-update-the-seconds-in-real-time

Reply:
You can keep the time in the state as total number of seconds, and in the render function you can transform those seconds into that mm:ss format: ```seconds / 60 + " : " +  seconds % 60```

Reply:
Also the time should have a bounding box, with different colors for current player & oponent.

Reply:
The name should also slightly change the color based on the turn. You can pass the turn property as a boolean prop: ```currentTurn: true/false```. We will control this from the parent.

Reply:
You can find the colors in the sketch file, by clicking the text/view element